chord_metadata_service/mohpackets/api_discovery.py

```diff
@@ -120,3 +142,196 @@ class DiscoveryBiomarkerViewSet(DiscoveryMixin, BaseBiomarkerViewSet):
+            fields={
+                "gender_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def gender_count(_request):
+    """
+    Return the count for every gender in the database.
+    """
+    genders = SampleRegistration.objects.values_list("gender", flat=True)
+    return Response(count_terms(genders))
+
+
+@extend_schema(
+    description="MoH cancer types count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_cancer_type_count_response",
+            fields={
+                "cancer_type_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def cancer_type_count(_request):
+    """
+    Return the count for every cancer type in the database.
+    """
+    cancer_types = Donor.objects.values_list("primary_site", flat=True)
+    return Response(count_terms(cancer_types))
+
+
+@extend_schema(
+    description="MoH Treatments type count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_treatment_type_count_response",
+            fields={
+                "treatment_type_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def treatment_type_count(_request):
+    """
+    Return the count for every treatment type in the database.
+    """
+    treatment_types = Treatment.objects.values_list("treatment_type", flat=True)
+    return Response(count_terms(treatment_types))
+
+
+@extend_schema(
+    description="MoH Diagnosis age count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_diagnosis_age_count_response",
+            fields={
+                "age_range_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def diagnosis_age_count(_request):
+    """
+    Return the count for age of diagnosis in the database.
+    """
+    # Find the earliest diagnosis date per donor
+    diagnosis_dates = PrimaryDiagnosis.objects.values("submitter_donor_id", "date_of_diagnosis")
+    min_dates = {}
+    for date in diagnosis_dates:
+        donor = date["submitter_donor_id"]
+        cur_date = dt.strptime(date["date_of_diagnosis"], "%Y-%m")
+        if donor not in min_dates.keys():
+            min_dates[donor] = cur_date
+        else:
+            if cur_date < min_dates[donor]:
+                min_dates[donor] = cur_date
+
+    # Calculate donor's age of diagnosis
+    birth_dates = Donor.objects.values("submitter_donor_id", "date_of_birth")
+    birth_dates = {date["submitter_donor_id"]: date["date_of_birth"] for date in birth_dates}
+    ages = {}
+    for donor, diagnosis_date in min_dates.items():
+        if donor in birth_dates.keys():
+            birth_date = dt.strptime(birth_dates[donor][:-3], "%Y-%m")
```

Comment:
I have the synthetic data in format yyyy-mm-dd but it will probably just yyyy-mm. You can assume it yyyy-mm and I will fix the synthetic data